spec/poros/movie_spec.rb

```diff
@@ -1,18 +1,18 @@
-require "rails_helper"
```

Comment:
This was the previous poro spec, I think the updates are only for rubocop

Reply:
That is correct.